server/src/test/java/org/elasticsearch/test/InternalSettingsPlugin.java

```diff
@@ -52,7 +51,6 @@
                 INDEX_CREATION_DATE_SETTING,
                 PROVIDED_NAME_SETTING,
                 TRANSLOG_RETENTION_CHECK_INTERVAL_SETTING,
-                IndexService.GLOBAL_CHECKPOINT_SYNC_INTERVAL_SETTING,
```

Comment:
This is already registered in https://github.com/crate/crate/blob/92f0909cf182da8fb560af142ad05492bc04bc2c/server/src/main/java/org/elasticsearch/common/settings/IndexScopedSettings.java#L119 
The setting is internal and should not be registered there as mentioned here
https://github.com/crate/crate/blob/8322e52ac7a26bdbe90e5c9bee9a3428c6c86794/server/src/main/java/org/elasticsearch/index/IndexService.java#L106
However, i can clean that up in a follow up.

Reply:
https://github.com/crate/crate/pull/10652